packages/utils/tsconfig.lib.json

```diff
@@ -0,0 +1,22 @@
+{
+    "compileOnSave": false,
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
```

Comment:
please change target to `esnext` we are going to do this change everywhere, since code is polyfilled and complied with webpack anyway

Reply:
:+1: